core/variant.h

```diff
@@ -503,6 +503,25 @@ const Variant::ObjData &Variant::_get_obj() const {
 	return *reinterpret_cast<const ObjData *>(&_data._mem[0]);
 }
 
-String vformat(const String &p_text, const Variant &p1 = Variant(), const Variant &p2 = Variant(), const Variant &p3 = Variant(), const Variant &p4 = Variant(), const Variant &p5 = Variant());
```

Comment:
You're not removing the implementation of `vformat()` in the `.cpp`, which should now become redundant? Not sure how this even compiles.